lib/modules/userInfo.js

```diff
@@ -209,6 +212,27 @@ async function showUserInfo(card) {
 		userHTML += string.escape`<div class="${isHighlight ? 'blueButton' : 'redButton'}" id="highlightUser" data-userid="${data.id}">${isHighlight ? i18n('userInfoHighlight') : i18n('userInfoUnhighlight')}</div>`;
 	}
 
+	if (data.name) {
```

Comment:
Note that `data.name` always is truish at this stage, so this conditional is not needed.